### ISSUE TITLE
Feedback for “Customers”

### ISSUE THREAD

**Issue by sshaw** [bigcommerce/docs#620]
https://developer.bigcommerce.com/docs/rest-management/customers#get-all-customers: 

> date_created:min in query - string
> Filter items by date created for example, 2024-05-14T09:34:00 or 2024-05-14. Returns metafields created after this date.

It says "Returns metafields" but it returns customers. 

Also worth noting that the date string must not be URI nor form encoded. For example:

```
2024-10-19T20%3A00%3A00-0400
```

Won't work, one must use: 

```
2024-10-19T20:00:00-0400
```

Maybe a bug but if this is an API-wide rule (I just checked with customers) should probably be mentioned here: https://developer.bigcommerce.com/docs/start/about/common-query-params

**Comment by bc-traciporter** [comment]
Hello @sshaw You found a mistake I made while cutting and pasting the `date_created:min` description.  I fixed this in [PR 621](https://github.com/bigcommerce/docs/pull/621).

As far as the date format examples provided, the examples work.
<img width="827" alt="Screenshot 2024-10-31 at 10 01 58 AM" src="https://github.com/user-attachments/assets/ec49a9a9-8425-41f3-821e-8cf4aab38b7b">

**Comment by sshaw** [comment]
> As far as the date format examples provided, the examples work.

Yes but the date string must not be URI nor form encoded. You can encoded `date_created:min` to `date_created%3Amin` and it still works. You can encode any of the parameters the API endpoint accepts. But you must not encode the date value else the request does not return the proper results.
